velox/type/Timestamp.h

```diff
@@ -69,6 +69,9 @@ struct TimestampToStringOptions {
   Mode mode = Mode::kFull;
 };
 
+// Returns the max size of timestamp string.
+std::string::size_type getCapacity(const TimestampToStringOptions& options);
```

Comment:
This function name is a bit ambiguous, can we use `getMaxLength`?

Reply:
Changed to getMaxStringLength.